src/jit/protojit/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+project(protojit)
```

Comment:
I have just added it for you, it will get propagated in a couple of minutes ... you do not need to worry about it.